tpu-client/src/nonblocking/tpu_client.rs

```diff
@@ -503,17 +503,55 @@ where
         Self::new_with_connection_cache(rpc_client, websocket_url, config, connection_cache).await
     }
 
+    /// Try to create LeaderTpuService
+    /// Retries until successful or timeout is reached
+    async fn try_create_leader_tpu_service(
+        rpc_client: Arc<RpcClient>,
+        websocket_url: &str,
+        exit: Arc<AtomicBool>,
+        timeout_seconds: u64,
```

Comment:
It is much better to use `Duration` rather than a `u64`.
In general, basic types should be avoided, as they carry less information and have fewer constraints attached to them.
It is easier to misuse them, with no or little benefit.

```suggestion
        timeout: Duration,
```